### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,129 +3,123 @@
...
-
-Developed based on [Snapdrop](https://github.com/RobinLinus/snapdrop)
+File sharing on your local network that works on all platforms.
+
+- A multi-platform AirDrop-like solution that works.
+  - Send images, documents or text via peer-to-peer connection to devices on the same local network.
+- Internet transfers
+  - Join temporary public rooms to transfer files easily over the Internet.
+- Web-app 
+  - Works on all devices with a web browser (that supports HTML5, CSS 3, JS, and WebRTC).
```
Comment (schlagmichdoch) [inline README.md:26]: Sounds much better but I think the information in brackets is too detailed for Users which are addressed by this paragraph. I would simply do: "Works on almost all devices with a web browser."

File: README.md [hunk 2]
```diff
@@ -3,129 +3,123 @@
...
-<details><summary>Click to expand</summary>
+## Differences to the [Snapdrop](https://github.com/RobinLinus/snapdrop) it is based on
+<details><summary>List view</summary>
 
-### Paired Devices and Public Rooms - Internet Transfer
-* Transfer files over the internet between paired devices or by entering temporary public rooms.
-* Connect to devices in complex network environments (public Wi-Fi, company network, Apple Private Relay, VPN etc.).
+### Paired Devices and Public Rooms — Internet Transfer
+* Transfer files over the Internet between paired devices or by entering temporary public rooms.
+* Connect to devices in complex network environments (public Wi-Fi, company network, Apple private relay, VPN, etc.).
```
Comment (schlagmichdoch) [inline README.md:37]: It's actually neither of ours. It's written "iCloud Private Relay" (see https://support.apple.com/en-us/102602)

File: README.md [hunk 3]
```diff
@@ -3,129 +3,123 @@
...
 * Connect to devices on your mobile hotspot.
-* Devices outside your local network that are behind a NAT are connected automatically via the PairDrop TURN server.
+* Devices outside of your local network that are behind a NAT are autop-connected via the PairDrop TURN server.
```
Comment (schlagmichdoch) [inline README.md:39]: Fix "auto**p**-connected"

File: README.md [hunk 4]
```diff
@@ -3,129 +3,123 @@
...
 * Connect to devices on your mobile hotspot.
-* You will always discover devices on your local network. Paired devices and devices in the same public room are shown additionally.
+* Devices that are paired devices, in the same public room, or on your local network are shown.
```
Comment (schlagmichdoch) [inline README.md:41]: "Devices" is twice. Also, I think this changes the meaning of the sentence. Maybe we could do "Devices discover each other if they are either paired, in the same public room, or on the same local network."

File: README.md [hunk 5]
```diff
@@ -3,129 +3,123 @@
...
-### Other changes
-* Change your display name permanently to easily differentiate your devices
-* [Paste files/text and choose the recipient afterwords ](https://github.com/RobinLinus/snapdrop/pull/534)
+* [Send files directly from the context menu on Windows](docs/how-to.md#send-files-directly-from-context-menu-on-windows)
+* [Send directly from the "Share" menu on iOS](docs/how-to.md#send-directly-from-share-menu-on-ios)
+* [Send directly from the "Share" menu on Android](docs/how-to.md#send-directly-from-share-menu-on-android)
+* [Send directly via the command-line interface](docs/how-to.md#send-directly-via-command-line-interface)
+
+### Other Changes
+* Change your display name to easily differentiate your devices.
```
Comment (schlagmichdoch) [inline README.md:74]: We could also use "persistently". Same problem is mentioned for the notifications (see also https://github.com/schlagmichdoch/PairDrop/discussions/152)

File: README.md [hunk 6]
```diff
@@ -3,129 +3,123 @@
...
+* Switch theme back to auto/system after dark or light mode is on
 * Node-only implementation (Thanks [@Bellisario](https://github.com/Bellisario))
-* Automatic restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
+* Auto-restart on error (Thanks [@KaKi87](https://github.com/KaKi87))
 * Lots of stability fixes (Thanks [@MWY001](https://github.com/MWY001) [@skiby7](https://github.com/skiby7) and [@willstott101](https://github.com/willstott101))
 * To host PairDrop on your local network (e.g. on Raspberry Pi): [All peers connected with private IPs are discoverable by each other](https://github.com/RobinLinus/snapdrop/pull/558)
-* When hosting PairDrop yourself you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
-* Built-in translations
+* When hosting PairDrop yourself, you can [set your own STUN/TURN servers](docs/host-your-own.md#specify-stunturn-servers)
+* Translations.
```
Comment (schlagmichdoch) [inline README.md:86]: I mentioned the "built-in" as it was also tried to add translations via the [snapdrop-android](https://github.com/fm-sys/snapdrop-android) app via JavaScript injections. Built-in translations are more stable, obviously.

File: README.md [hunk 7]
```diff
@@ -3,129 +3,123 @@
...
-## PairDrop is built with the following awesome technologies:
-* Vanilla HTML5 / ES6 / CSS3 frontend
-* [WebRTC](http://webrtc.org/) / [WebSockets](http://www.websocket.org/)
-* [NodeJS](https://nodejs.org/en/) backend
-* [Progressive Web App](https://wikipedia.org/wiki/Progressive_Web_App)
+## Built with the following awesome technologies:
+* Vanilla HTML5 / JS ES6 / CSS 3 frontend
+* [WebRTC](http://webrtc.org/) / WebSockets
+* [Node.js](https://nodejs.org/en/) backend
+* [Progressive web app](https://wikipedia.org/wiki/Progressive_Web_App)
```
Comment (schlagmichdoch) [inline README.md:97]: The Wikipedia page was redirected. I like their naming: "Progressive web application (PWA)"

File: README.md [hunk 8]
```diff
@@ -3,129 +3,123 @@
...
-* [Weblate](https://weblate.org/) Web based localization tool
-
-Have any questions? Read our [FAQ](docs/faq.md).
+* [cyrb53](https://github.com/bryc/code/blob/master/jshash/experimental/cyrb53.js) super-fast hash function
+* [Weblate](https://weblate.org/) web-based localization tool
 
-You can [host your own instance with Docker](docs/host-your-own.md).
+[FAQ](docs/faq.md)
 
+[Host your own instance with Docker](docs/host-your-own.md).
```
Comment (schlagmichdoch) [inline README.md:105]: Much clearer, thanks! Maybe we could do "Host your own instance with Docker or Node.js"

File: README.md [hunk 9]
```diff
@@ -3,129 +3,123 @@
...
+## Support
 <a href="https://www.buymeacoffee.com/pairdrop" target="_blank">
-<img src="https://cdn.buymeacoffee.com/buttons/v2/default-yellow.png" alt="Buy Me A Coffee" style="height: 60px !important;width: 217px !important;" >
+<img src="https://cdn.buymeacoffee.com/buttons/v2/default-yellow.png" alt="Buy me a coffee" style="height: 60px !important;width: 217px !important;" >
 </a>
 
-PairDrop is free and always will be.
-Still, we have to pay for the domain and the server.
+PairDrop is libre, and always will be. \
+I footed the bill for the domain and the server, and you can help create great softeare by supporting me. \
```
Comment (schlagmichdoch) [inline README.md:113]: Hm.. I wanted to explicitly mention, that I the funds that are donated are mainly used to pay the server and the domain. Do you think this is still clear enough?

**Review by schlagmichdoch — CHANGES_REQUESTED** [review]
Thanks for your recurring contribution! I just added some comments and made all content links and image sources relative.

**Comment by schlagmichdoch** [comment]
Hey @comradekingu,
I have since introduced a new version `v1.10.0`. I would like to include your changes. Would you have a look at the review? :)

**Comment by schlagmichdoch** [comment]
Friendly bump @comradekingu :)

File: README.md (hunk 5 above)
Comment (comradekingu) [reply README.md:74]: Not sure how that makes anything easier to understand or where to add it.

File: README.md (hunk 6 above)
Comment (comradekingu) [reply README.md:86]: I think "translations" this is a long-term solution.

File: README.md (hunk 9 above)
Comment (comradekingu) [reply README.md:113]: Yes. Paying for servers isn't something people want to buy.

File: README.md [hunk 10]
```diff
@@ -3,133 +3,127 @@
...
+  - Join temporary public rooms to transfer files easily over the Internet.
+- Web-app 
+  - Works on all devices with a modern web-browser.
+ 
+Send a file from your phone to your laptop?
+<br>Share photos in original quality with friends using Android and iOS?
+<br>Share private files peer-to-peer between Linux systems?
 
-Developed based on [Snapdrop](https://github.com/RobinLinus/snapdrop)
+<img src="docs/pairdrop_screenshot_mobile.gif" alt="Gif of Screenshots that show PairDrop in use" style="width: 300px">
```
Comment (comradekingu) [inline README.md:32]:
```suggestion
<img src="docs/pairdrop_screenshot_mobile.gif" alt="Screenshot GIF showing PairDrop in use" style="width: 300px">
```

**Comment by comradekingu** [comment]
@schlagmichdoch Thank you for the reminder. Good to go now I should think.

**Review by schlagmichdoch — APPROVED** [review]

**Pull request merged**